.github/workflows/release.yml

```diff
@@ -0,0 +1,316 @@
+              shell: bash
+              run: echo "${{ github.workspace }}\mingw64\bin" >> $GITHUB_PATH
+
+            - name: Generate Import Libs for MinGW
+              shell: pwsh
+              run: |
+                  New-Item -ItemType "directory" -PATH "CSFML-install\lib\gcc"
+                  New-Item -ItemType "directory" -PATH "CSFML-install\lib\msvc"
+                  Move-Item -Path "CSFML-install\lib\*.lib" -Destination "CSFML-install\lib\msvc"
+
+                  Push-Location CSFML-install\lib\gcc
+                  & gendef.exe "..\..\bin\csfml-graphics-2.dll"
+                  & dlltool.exe -d "csfml-graphics-2.def" -D "..\..\bin\csfml-graphics-2.dll" -l "libcsfml-graphics.a"
+                  Remove-Item -Path "csfml-graphics-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-window-2.dll"
+                  & dlltool.exe -d "csfml-window-2.def" -D "..\..\bin\bin\csfml-window-2.dll" -l "libcsfml-window.a"
+                  Remove-Item -Path "csfml-window-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-audio-2.dll"
+                  & dlltool.exe -d "csfml-audio-2.def" -D "..\..\bin\bin\csfml-audio-2.dll" -l "libcsfml-audio.a"
+                  Remove-Item -Path "csfml-audio-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-network-2.dll"
+                  & dlltool.exe -d "csfml-network-2.def" -D "..\..\bin\bin\csfml-network-2.dll" -l "libcsfml-network.a"
+                  Remove-Item -Path "csfml-network-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-system-2.dll"
+                  & dlltool.exe -d "csfml-system-2.def" -D "..\..\bin\bin\csfml-system-2.dll" -l "libcsfml-system.a"
+                  Remove-Item -Path "csfml-system-2.def"
+                  Pop-Location
+
+                  New-Item -ItemType "directory" -PATH "install"
+                  Move-Item -Path "CSFML-install\*" -Destination "install\CSFML"
+
+            - name: Upload Artifact
+              uses: actions/upload-artifact@v4
+              with:
+                  name: CSFML-windows-64-bit
+                  path: install/CSFML
+
+    macos-x64:
+        name: macOS x64
+        runs-on: macos-12
+
+        steps:
+            - name: Checkout SFML
+              uses: actions/checkout@v4
+              with:
+                  repository: SFML/SFML
+                  ref: 2.6.1
+                  path: SFML
+
+            - name: Checkout CSFML
+              uses: actions/checkout@v4
+              with:
+                  path: CSFML
+
+            - name: Build SFML
+              run: |
+                  cmake -S SFML -B SFML-build -G "Unix Makefiles" \
+                        -DCMAKE_C_COMPILER="/usr/bin/clang" \
+                        -DCMAKE_CXX_COMPILER="/usr/bin/clang++" \
+                        -DCMAKE_C_FLAGS="-stdlib=libc++" \
+                        -DCMAKE_CXX_FLAGS="-stdlib=libc++" \
```

Comment:
These choices for generator, compiler, and standard library are already the defaults on macOS so we can omit these arguments if we want.

Reply:
Hmmm I did get different results when I was playing with SDKs for the standard library and compiler, but I guess it's not needed for the standard case